project_task_description_template/models/project_task.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    task_template_id = fields.Many2one(
+        string="Template", comodel_name="project.task.template"
+    )
+
+    template_visible = fields.Boolean(compute="_compute_template_visible")
+
+    @api.depends("project_id", "stage_id")
+    def _compute_template_visible(self):
+        for rec in self:
+            if (
```

Comment:
I think this solution is not good. Better do like this:
```py
rec.template_visible = rec.stage_id and rec.project_id and rec.project_id.template_task_type_ids.ids and rec.stage_id.id in rec.project_id.template_task_type_ids.ids
```

Reply:
thanks, I'll fix it